BREAKING.md

```diff
@@ -30,6 +30,7 @@ IFluidResolvedUrl is now deprecated, all usages should move to IResolvedUrl inst
 -   [IResolvedUrl equivalent to IFluidResolvedUrl](#IResolvedUrl-equivalent-to-IFluidResolvedUrl)
 -   [garbage-collector and related items removed](#garbage-collector-and-related-items-removed)
 -   [GC interfaces removed from runtime-definitions](#gc-interfaces-removed-from-runtime-definitions)
+-   [Upgraded Typescript target to ES2020](#Upgraded-Typescript-target-to-ES2020)
```

Comment:
I don't think this should be in breaking.md until we actually bump client to use it.